### ISSUE TITLE
Blacklist value changes of certain attributes

### ISSUE THREAD

**Issue by jespr** [grosser/cia#12]
Your example in the README uses `crypted_password`. It might make sense to audit that the value of that field changed, but I don't think it makes sense to actually store the `old` and `new` value of that field in the audits table, as that would leak password values to another table than wherever the password is currently stored.

What do you say @grosser ?

**Comment by grosser** [comment]
maybe audit the first x characters so the diff is visible ... otherwise just got to find something that is not confusing to end-users that will see the metric ... or some plain 'old -> new'

**Comment by jespr** [comment]
Why would end-users care what their `crypted_password` is (let's just continue using that example). I would freak out if I saw that was logged in an audit table.

For other things, it might make sense to log the first few chars.

**Comment by grosser** [comment]
yeah true ... should be hidden from the user ... then it can be whatever
makes sense ...
